charts/newrelic-infrastructure/templates/controlplane/daemonset.yaml

```diff
@@ -79,10 +79,10 @@ spec:
             - name: NRI_KUBERNETES_VERBOSE
               value: "1"
             {{- end }}
-            {{- with .Values.controlPlane.extraEnv }}
```

Comment:
I guess you discussed the name already, *if not* are we sure that it worth to change this?